### ISSUE TITLE
 C3D 1.1 sports-1m model (conv3d_deepnetA_sport1m_iter_1900000) 

### ISSUE THREAD

**Issue by wjb123** [facebook/C3D#210]
Can you release your  C3D 1.1 sports-1m model (conv3d_deepnetA_sport1m_iter_1900000), and I need to use the C3D to retrain my own network.

**Comment by InnovArul** [comment]
There is a pretrained model available with C3D-v1.1 if that's what you are looking for. Please look into this file for the download link of the model.

https://github.com/facebook/C3D/blob/master/C3D-v1.1/examples/c3d_ucf101_feature_extraction/feature_extraction.sh

**Comment by dutran** [comment]
But that is a different architecture based on Resnet. Ffor C3D-VGG, I have to convert it into the new format later. Will keep this issue open until I share with you the converted model.

**Comment by wjb123** [comment]
Hope you release it soon.

**Comment by bityangke** [comment]
@dutran Hi, dutran, how can I fine-tuning on my data using C3D-ResNet18-Sports1M?
Thank you in advance!

**Comment by dutran** [comment]
@bityangke try the example at https://github.com/facebook/C3D/tree/master/C3D-v1.1/examples/c3d_ucf101_finetuning

**Comment by bityangke** [comment]
Thank you very much！

**Comment by bityangke** [comment]
@dutran  Hi, bother you again. 
But I can not find 'c3d_resnet18_ucf101_r2_ft_iter_20000.caffemodel' that you have fine-tuned on UCF-101.
Have you shared it online?

**Comment by dutran** [comment]
@bityangke try this https://www.dropbox.com/s/bf5z2jw1pg07c9n/c3d_resnet18_ucf101_r2_ft_iter_20000.caffemodel?dl=0

**Comment by bityangke** [comment]
@dutran Thank you very much !

**Comment by vivoutlaw** [comment]
@dutran : have you been able to convert the original C3D model into the new format for C3DV1.1? Thanks!

**Comment by dutran** [comment]
@vivoutlaw Opps, I closed it by accident. I haven't done this yet.

**Comment by rezvannzri** [comment]
@dutran :I'd appreciate  you if you could release your C3D 1.1 sports-1m model (conv3d_deepnetA_sport1m_iter_1900000) .

**Comment by samiksome** [comment]
@rezvannzri I had converted the v1.0 weights to v1.1 format a while back. You can use them if you want. 
However, I did not convert the old mean file.

https://github.com/samiksome/C3D-weights

**Comment by rezvannzri** [comment]
@samiksome : Thank you so much!

**Comment by kasparov92** [comment]
@dutran  @InnovArul is this model (c3d_resnet18_ucf101_r2_ft_iter_20000.caffemodel) trained on ucf101 or trained on sports1m and fine tuned on UCF101

**Comment by dutran** [comment]
c3d_resnet18_ucf101_r2_ft_iter_20000.caffemodel is finetune on UCF101 split1 (after being trained Sports1M)

**Comment by kasparov92** [comment]
@dutran  I am using the c3d_resnet18_ucf101_r2_ft_iter_20000.caffemodel for feature extraction and extract only  the prob. In the output, I expected probability for each of the 101 classes but instead, I find probs for 487 classes, which must be the classes of sports1M. Is something wrong with this model?

**Comment by dutran** [comment]
you used the wrong prototxt file to extract feature.

**Comment by kasparov92** [comment]
Yeah, you are right, I used the one in the `c3d_ucf101_feature_extraction` folder, shall I change the line at the end to make the num of classes 101 or shall I use the prototext in the `c3d_ucf101_finetuning` folder called `train_resnet18_r2.prototxt`

**Comment by yzldw333** [comment]
@samiksome It works, thank u.

**Comment by xiaozeyuan** [comment]
@dutran  I am using the conv3d_deepnetA_sport1m_iter_1900000 tfor feature extraction,but here is an error "Check failed: num_axes() <= 4 (5 vs. 4) Cannot use legacy accessors on Blobs with > 4 axes." in Relu layer.How to solve it?

**Comment by xiaozeyuan** [comment]
@dutran I am using the conv3d_deepnetA_sport1m_iter_1900000 for feature extraction,but here is an error "Check failed: num_axes() <= 4 (5 vs. 4) Cannot use legacy accessors on Blobs with > 4 axes." in Relu layer.How to solve it?

**Comment by samiksome** [comment]
@xiaozeyuan I believe you are trying to use old C3D v1.0 weights with C3D v1.1 code.

**Comment by xiaozeyuan** [comment]
@samiksome I download the weights from "https://github.com/samiksome/C3D-weights",and the prototxt file  i used is c3d_sport1m_feature_extractor_frm_v1.1.prototxt

**Comment by xiaozeyuan** [comment]
@samiksome I download the weights from "https://github.com/samiksome/C3D-weights", and the prototxt file i used is c3d_sport1m_feature_extractor_frm_v1.1.prototxt

**Comment by samiksome** [comment]
@xiaozeyuan are you using this with C3D v1.1 or C3D v1.0?

**Comment by xiaozeyuan** [comment]
@samiksome  C3D v1.1

**Comment by samiksome** [comment]
@xiaozeyuan I just ran it with C3D v1.1 and it ran fine. You are using both weights and prototxt from "https://github.com/samiksome/C3D-weights" and C3D v1.1 `extract_image_features.bin` right?

**Comment by xiaozeyuan** [comment]
@samiksome  I do use these files to extract C3D features.The puzzling problem made me confused.

**Comment by Darren1988** [comment]
@dutran Hi, just want to use the C3D to extract some features from the video frames. May I ask has the model 'conv3d_deepnetA_sport1m_iter_1900000' been released or not? 
Many thanks for your time.

**Comment by Darren1988** [comment]
@samiksome Hi, I found that "you had converted the v1.0 weights to v1.1 format a while back." May I ask what is the difference between the v1.0 weights and v1.1 format? Can I use your converted weights with the v1.0 code?

**Comment by samiksome** [comment]
@Darren1988 I believe v1.0 and v1.1 use different versions of caffe which causes the weights to be incompatible. weights saved in v1.0 cannot be used v1.1 and vice-versa. As such the converted weights would only work with v1.1 code.

**Comment by Darren1988** [comment]
@samiksome Thanks a lot for your prompt reply and explanation. I think the weights you mentioned is the exact weights I wanted to extract my features.

**Comment by Zakia13** [comment]
Hello. i'm newbie to Deep learning. I want to use C3D pretrained model as a feature extractor and extract features from each 16 frame long clips.  I'm using this code https://github.com/hx173149/C3D-tensorflow. But i'm confused how to use it. Kindly guide me on this. Thanks.

**Comment by xuyunlu1030** [comment]
I tried both the C3D 1.1 sports-1m pretrained model and UCF101 finetuned model, but the loss does not decrease (the same as training from the scratch). See Issue: https://github.com/facebook/C3D/issues/351
I am confused about the problem. Looking forward your suggestions.

**Comment by YoungerGao** [comment]
> @rezvannzri I had converted the v1.0 weights to v1.1 format a while back. You can use them if you want.
> However, I did not convert the old mean file.
> 
> https://github.com/samiksome/C3D-weights

where can i find the covert tools
